CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [#6032](https://github.com/apache/trafficcontrol/issues/6032) Add t3c setting mode 0600 for secure files
 - [#6405](https://github.com/apache/trafficcontrol/issues/6405) Added cache config version to all t3c apps and config file headers
 - Traffic Vault: Added additional flag to TV Riak (Deprecated) Util
+- ATC Collaborators: Added a GitHub Action to grant the GitHub Triage role for a month to contributors who closed 2 Issues in the past month
```

Comment:
This isn't actually a change to the software that's present in any given version. Collaborators will have permissions on GitHub, and can e.g. triage backport issues for old versions. It's not a change of which people using the software need to be made aware. Which is to say, I don't think this needs a changelog entry.

Reply:
Removed changelog entry in 3dd11ef81a